### PR TITLE
[hotfix] [docs] Fix typo in documentation

### PR DESCRIPTION
**Pull request by zzchun**
## What is the purpose of the change

This pull request displays the `Connectors` and `Formats` tables in `Connect to External Systems` page. 

For an unstable version, these tables can not be seen in the page but we described them, which is confusing.  Therefore, I removed the Stable checking logic and add a Note to make it more readable. 

What's more, this PR also fixes a typo in `Time Attributes` documentation.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/dev/table/connect.md [hunk 1]
```diff
@@ -58,11 +56,7 @@ The following table list all available connectors and formats. Their mutual comp
 | JSON              | `flink-json`                 | [Download](http://central.maven.org/maven2/org/apache/flink/flink-json/{{site.version}}/flink-json-{{site.version}}-sql-jar.jar) |
 | Apache Avro       | `flink-avro`                 | [Download](http://central.maven.org/maven2/org/apache/flink/flink-avro/{{site.version}}/flink-avro-{{site.version}}-sql-jar.jar) |
 
-{% else %}
-
-This table is only available for stable releases.
-
-{% endif %}
```
Comment (klion26) [inline docs/dev/table/connect.md]: Why did you remove the if/else?

Comment (twalthr) [reply docs/dev/table/connect.md]: You are using docs of an unstable release. Why should this be confusing? Please undo these changes.

Comment (zzchun) [reply docs/dev/table/connect.md]: @twalthr I have recovered the changes, can you help me to have a review, thank you!
